Sources/Sentry/include/SentryOptions.h

```diff
@@ -54,6 +54,10 @@ SENTRY_NO_INIT
  */
 @property(nonatomic, copy) NSNumber *enabled;
 
+/**
+ * How many breadcrumbs do you want to keep in memory?
+ * We have a hard cap limit on 250. Default is 100.
```

Comment:
I believe "hard limit" and "cap" are synonymous  
```suggestion
 * We have a hard limit of 250. Default is 100.
```

This _hard limit_ is enforced by Sentry itself?